src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const discountAfterTwoDays = 20;
+  const discountAfterSixDays = 50;
```

Comment:
```suggestion
  const smallDiscount = 20;
  const largeDiscount = 50;
```
why after two or six? what if we change our business requirements and there will be 3 days and 5 days? Then we should update it everywhere in code, it's not ok